graphql_test.go

```diff
@@ -245,6 +247,63 @@ func TestBasic(t *testing.T) {
 	})
 }
 
+type testNilInterfaceResolver struct{}
+
+func (r *testNilInterfaceResolver) A() interface{ Z() int32 } {
```

Comment:
Any reason for inlining the interface instead of defining it as a top-level type?